docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,244 @@
+.. _guide_scaleway:
+
+***************************
+Using Scaleway with Ansible
+***************************
+
+.. _scaleway_introduction:
+
+Introduction
+============
+
+`Scaleway <https://scaleway.com>`_ is a cloud provider supported by Ansible, version 2.6 or higher via a dynamic inventory plugin and modules.
+Those modules are:
+
+- :ref:`scaleway_sshkey_module`: adds a public SSH key from a file or value to the Packet infrastructure. Every subsequently-created device will have this public key installed in .ssh/authorized_keys.
+- :ref:`scaleway_compute_module`: manages servers on Scaleway. You can use this module to create, restart and delete servers.
+- :ref:`scaleway_volume_module`: manages volumes on Scaleway.
+
+.. note::
+   This guide assumes you are familiar with Ansible and how it works.
+   If you're not, have a look at :ref:`ansible_documentation` before getting started.
+
+.. _scaleway_requirements:
+
+Requirements
+============
+
+The Scaleway modules and inventory script connect to the Scaleway API using `Scaleway REST API <https://developers.scaleway.com>`_.
```

Comment:
this url didn't work for me.. but https://developer.scaleway.com did...

Reply:
Fixed